### PR TITLE
Project/afw display/brant

### PR DESCRIPTION
**Pull request by brantr**
I've created a brief tutorial to walk users through displaying a Twinkles calexp image using lsst.afw.display. This notebook is based on cells from the LowSurfaceBrightness notebook by @kadrlica. The notebook compiles and runs with the stack v16.

I'm sure this notebook could be extended to better exercise some of afw.display's functionality, but I thought it might be useful to pull this now, and then update with more examples at a later date.

### CODE REVIEW

File: Visualization/AFW_Display_Demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# **Demo of lsst.afw.display -- displaying images using the LSST DM Astronomical Framework library**\n",
+    "\n",
+    "**Owner:** Brant Robertson ([@brantr](https://github.com/LSSTScienceCollaborations/DMStackClub/issues/new?body=@brantr))  \n",
+    "**Level:** Introductory  \n",
```
Comment (drphilmarshall) [inline Visualization/AFW_Display_Demo.ipynb]: Nice! We should adopt this in our other notebooks too.

File: Visualization/AFW_Display_Demo.ipynb [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# **Demo of lsst.afw.display -- displaying images using the LSST DM Astronomical Framework library**\n",
+    "\n",
+    "**Owner:** Brant Robertson ([@brantr](https://github.com/LSSTScienceCollaborations/DMStackClub/issues/new?body=@brantr))  \n",
```
Comment (drphilmarshall) [inline Visualization/AFW_Display_Demo.ipynb]: We should make sure all our URLs say "StackClub" and not "DMStackClub"

File: Visualization/AFW_Display_Demo.ipynb [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+    "And let the kernel know that we're happy not to have some useful warnings printed during this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "warnings.simplefilter(\"ignore\", category=FutureWarning)   #prevent some helpful but ancillary warning messages from printing during LSST DM Release v16 calls\n",
```
Comment (drphilmarshall) [inline Visualization/AFW_Display_Demo.ipynb]: This comment about v16 will go stale - better to just say "prevent some ... during LSST DM calls"

File: Visualization/AFW_Display_Demo.ipynb [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Step 2) Importing Data to Visualize**\n",
+    "\n",
+    "To plot an image to the screen, we must first load some data. In this tutorial, we will use the `Twinkles` simulated images available in the StackClub data repository. These data sit in the data directory `/project/shared/data/Twinkles_subset/output_data_v2` and contain a set of data produced in generating a calibrated exposure by the DM Stack. These data are organized in a structure that enables a DM Stack `Butler` instance to be generated and provide access to a single filter image (in this case `r` band), a specific detector raft (2,2), a specific sensor in the raft (`1,1`) and a specific visit (in this case, 235 -- note only one band is available per visit in this example).\n",
+    "\n",
+    "Once we define a string that contains the data directory, we start the `Butler` instance using the `lsst.daf.persistence` library alias `dafPersist` and its function `Butler()`. The function `Butler()` takes as an argument a string containing the data directory we wish to access. Running the cell may take a few moments."
```
Comment (drphilmarshall) [inline Visualization/AFW_Display_Demo.ipynb]: I think `Butler` is a class, not a function - I'll submit a small PR with better wording.

File: Visualization/AFW_Display_Demo.ipynb [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+   "source": [
+    "method_list = [func for func in dir(afw_display) if callable(getattr(afw_display, func))]\n",
+    "print(method_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you'd like to learn more about any given function, please see the [`lsst.afw.display` source code](https://github.com/lsst/afw/tree/master/python/lsst/afw/display)."
```
Comment (drphilmarshall) [inline Visualization/AFW_Display_Demo.ipynb]: We can also recommend the notebook `help()` function for showing the docstrings to the above functions - I'll include this in my PR too.

File: Visualization/AFW_Display_Demo.ipynb (hunk 1 above)
Comment (brantr) [reply Visualization/AFW_Display_Demo.ipynb]: Great, thanks.

File: Visualization/AFW_Display_Demo.ipynb (hunk 2 above)
Comment (brantr) [reply Visualization/AFW_Display_Demo.ipynb]: Fixed, thanks.

File: Visualization/AFW_Display_Demo.ipynb (hunk 3 above)
Comment (brantr) [reply Visualization/AFW_Display_Demo.ipynb]: Fixed, thanks.

**Comment by brantr** [comment]
I've made the changes requested by @drphilmarshall . If further changes are needed, please let me know.

File: Visualization/AFW_Display_Demo.ipynb [hunk 6]
```diff
@@ -0,0 +1,323 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now set to display the image. To do this, we:\n",
+    "\n",
+    "* First create a `matplotlib.pyplot` figure using `plt.figure()` -- this will be familiar to anyone with experience using `matplotlib`.\n",
+    "* Then create an alias to the `lsst.afw.display.Display` method that will allow us to display the data to the screen.  This alias will be called `afw_display`.\n",
```
Comment (kadrlica) [inline Visualization/AFW_Display_Demo.ipynb]: I think that what you are actually doing is using the `afw_display` variable to persist the `lsst.afw.display.interface.Display` object that is returned by `lsst.afw.display.Display`.

**Pull request merged**